tests/test_utility_sim2sumo.py

```diff
@@ -0,0 +1,274 @@
+"""Test utility ecl2csv"""
+import sys
+import os
+from time import sleep
+import logging
+from pathlib import Path
+import pandas as pd
+import pyarrow as pa
+import pytest
+from sumo.wrapper import SumoClient
+from fmu.sumo.utilities import sim2sumo
+from fmu.sumo.uploader import CaseOnDisk, SumoConnection
+
+
+REEK_ROOT = Path(__file__).parent / "data/reek"
+REAL_PATH = "realization-0/iter-0/"
+REEK_REAL = REEK_ROOT / REAL_PATH
+REEK_BASE = "2_R001_REEK"
+REEK_ECL_MODEL = REEK_REAL / "eclipse/model/"
+REEK_DATA_FILE = REEK_ECL_MODEL / f"{REEK_BASE}-0.DATA"
+CONFIG_OUT_PATH = REEK_REAL / "fmuconfig/output/"
+CONFIG_PATH = CONFIG_OUT_PATH / "global_variables.yml"
+
+
+logging.basicConfig(
+    level=logging.info, format=" %(name)s :: %(levelname)s :: %(message)s"
+)
+LOGGER = logging.getLogger(__file__)
+
+
+def test_submodules_dict():
+    """Test generation of submodule list"""
+    sublist, submods = sim2sumo._define_submodules()
+    LOGGER.info(submods)
```

Comment:
Put the logging in the source code, not in the test? Or do we want logging output also for the tests?

Reply:
We want some logging output, just when developing.